hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -230,25 +245,53 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
         return pd.DataFrame(data=data, columns=columns)
 
     def format_status(self) -> str:
-        cdef:
-            list lines = []
-            list warning_lines = []
-            dict tracked_maker_orders = {}
-            LimitOrder typed_limit_order
+        lines = []
+        warning_lines = []
+        tracked_maker_orders = {}
 
         # Go through the currently open limit orders, and group them by market pair.
         for _, limit_order in self.active_limit_orders:
             typed_limit_order = limit_order
-            market_pair = self._market_pair_tracker.c_get_market_pair_from_order_id(typed_limit_order.client_order_id)
+            market_pair = self._market_pair_tracker.get_market_pair_from_order_id(typed_limit_order.client_order_id)
             if market_pair not in tracked_maker_orders:
                 tracked_maker_orders[market_pair] = {typed_limit_order.client_order_id: typed_limit_order}
             else:
                 tracked_maker_orders[market_pair][typed_limit_order.client_order_id] = typed_limit_order
 
         for market_pair in self._market_pairs.values():
+            if self.is_gateway_market(market_pair.taker):
+                is_gateway = True
+            else:
+                is_gateway = False
+
```

Comment:
I don't think this is necessary since we can just turn the next if in `if not self.is_gateway_market(market_pair.taker):`

Reply:
✅